helm_deploy/create-and-vary-a-licence/templates/activate-licences-cronjob.yaml

```diff
@@ -3,7 +3,7 @@ kind: CronJob
 metadata:
   name: activate-licences
 spec:
-  schedule: "0 3,12 * * *" # 3am and 12PM every day
+  schedule: "0 9,12,15, 18 * * *" # 9am, 12, 15, 18 every day
```

Comment:
A minor thing, could we change the comment here to be `# 9am, 12pm, 3pm, and 6pm every day`?
Just to make it more easily human-readable

Reply:
Done.